src/getCoinCombination.test.js

```diff
@@ -7,4 +7,28 @@ describe('getCoinCombination', () => {
     expect(getCoinCombination)
       .toBeInstanceOf(Function);
   });
+
+  it(`should be return '[1, 1, 0, 0]' when argument '6'`, () => {
+    const result = getCoinCombination(6);
+
+    expect(result).toEqual([1, 1, 0, 0]);
+  });
+
+  it(`should be return '[0, 0, 0, 2]' when argument '50'`, () => {
```

Comment:
The test description should be corrected to 'should return [0, 0, 0, 2] when argument is 50' for clarity and grammatical correctness.